modules/ppcp-wc-gateway/src/Checkout/DisableGateways.php

```diff
@@ -107,7 +107,7 @@ private function disable_all_gateways() : bool {
 	 */
 	private function needs_to_disable_gateways(): bool {
 		$wc_ajax = filter_input( INPUT_GET, 'wc-ajax', FILTER_SANITIZE_STRING ) ?? '';
-		if ( $wc_ajax === 'update_order_review' ) {
+		if ( $wc_ajax === 'update_order_review' && $this->session_handler->funding_source() !== 'paypal' ) {
```

Comment:
hm, will it still work correctly when using other buttons like Venmo, Pay Later?

Reply:
@AlexP11223 yup it wirks for other buttons, the fix was for not hidding other gateways after getting a WC validation error when using DCC payment. I used `update_order_review` to ensure we are on that context, but when coming from express checkout `update_order_review` also exist and hides the other gateways, that´s the reason I added the conditional to not execute it when is `paypal`.

Reply:
you´re right, it only handles PayPal button.